streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -436,6 +438,11 @@ void closeSuspended(boolean clean, RuntimeException firstException) {
         }
     }
 
+    @Override
+    public Map<TopicPartition, Long> checkpointedOffsets() {
+        return Collections.emptyMap();
```

Comment:
Similar here, we can throw unsupported exception and make sure `active task`'s function is never called in thread?